### DISCUSSION TITLE
SEO metadata tag for versions is missing a space between the number

### DISCUSSION THREAD

**Discussion by bnb** [npm/feedback#454] [Website] [answered]
For all modules, npm surfaces some nice additional metadata that helps provide relevant information on search engine results. It looks like one of these is missing a space, so the information that's surfaced is slightly incorrect.

I imagine this is just adding ` ` somewhere :)

![image](https://user-images.githubusercontent.com/502396/122704152-17347680-d221-11eb-88d2-89327d3030c1.png)

**Comment by lumaxis** [comment]
@bnb Do you remember how exactly you searched to produce these results? I'd like to try and repro them but haven't figured out how 🤔

**Reply by bnb** [reply]
@lumaxis I'm not certain what the original search was but I was able to reproduce it [here](https://www.google.com/search?q=express+npm+release&ei=dofTYOGVNca4-gT_4J3IBg&oq=express+npm+release&gs_lcp=Cgdnd3Mtd2l6EAMyBwghEAoQoAEyBwghEAoQoAEyBwghEAoQoAE6BwgAEEcQsAM6BwgAELADEEM6AggAOgYIABAWEB46BQghEKABSgQIQRgAULYJWMUQYKESaAFwAngAgAGpAogBmg-SAQMyLTiYAQCgAQGqAQdnd3Mtd2l6yAEKwAEB&sclient=gws-wiz&ved=0ahUKEwjhqJbuuq7xAhVGnJ4KHX9wB2kQ4dUDCBE&uact=5). It's come up a number of times in the past month or so for me.

**Comment by MylesBorins** [comment, marked as answer]
hey @bnb we are tracking fixing this internally now with engineering. There are a couple reasons why this is complicated due to how Google indexes pages, but we have some novel solutions in mind.

**Reply by lumaxis** [reply]
@bnb As @MylesBorins said, we're looking in a few general ways for improving our Google search results but this specific issue should be fixed and resolve once Google re-indexes the page 🙏🏼

**Reply by bnb** [reply]
thank you @lumaxis!
